content/raw/chat/online_users.md

```diff
@@ -26,19 +26,26 @@ It's important for end users to know which community members are online or offli
 
 ## Use Cases
 
-### Setting state as online
+### Setting state as always online
 
-1. User selects state as `online`
+1. User selects state as `always online`
 2. System updates indicator to indicate online state (e.g a green circle)
 3. System indicates user is online in the member list of each community the user is a part of
-4. Other users see this user as online in the member list of each community the user is part of
+4. Other users see this user as online in the member list of each community the user is part of, as long as the last ping was less than 2 weeks ago
 
-### Setting state as offline
+### Setting state as inactive
 
-1. User selects state as `offline`
+1. User selects state as `inactive`
 2. System updates indicator to indicate offline state (e.g a grey circle)
 3. System indicates user is offline in the member list of each community the user is a part of
-4. Other users see this user as offline in the member list of each community the user is part of, within a period of up to 5 minutes
+4. Other users see this user as offline in the member list of each community the user is part of
+
+### Setting state as automatic
+
+1. User selects state as `set status automatically`
+2. System updates indicator to indicate online state (e.g a green circle)
+3. System indicates user is online in the member list of each community the user is a part of
```

Comment:
...*IF* the user is active within Status

Reply:
of course the act of setting the status setting to 'set status automatically' would by definition mean that the user is active at the point they select a status setting ;-)